schultz_a02b.js

```diff
@@ -0,0 +1,30 @@
+//Chris Schultz
+//schultz_a02b.js
+//Info 2124 WW
+//Thoendel
+//12-10-2019
+
+//declare gloabl variables
+let age = 37; //globally-scoped variable delared outside brackets per req b
```

Comment:
the assignment was to have 1 variable that supported block scope and one that doesn't - you have 2 let declarations that would support block - but you don't have any var declarations (which wouldn't support block)
-2
Great attention to detail with the semicolons.